.travis.yml

```diff
@@ -3,6 +3,8 @@ language: python
 python:
   - "2.7"
 
+cache: pip
+
```

Comment:
I wonder if it's worth including the `node_modules` directory as @chosak did in #2381.